### PR TITLE
Adding a .varci.yaml

### PR DESCRIPTION
**Pull request by lorenzo**
This may not be necessary for this repo as the amount of change requests is usually low, but is very cool nevertheless 💃 

This should at least help automate some of the tedium of maintaining the list and communicating with contributors.

### CODE REVIEW

**Comment by ADmad** [comment]
Let's give this a try :+1:

**Comment by dereuromark** [comment]
Do we need some docs/readme update or some contribution file explaining what to do here?

**Comment by jadb** [comment]
Woohoo!! 🚀 

@dereuromark docs can be found here for now: https://github.com/VarCI/support/blob/master/pages/documentation.md. Let me know if you had something else in mind and I will get it PR'd later today.

Thanks for giving it a shot!

**Comment by dereuromark** [comment]
I mean for people making a PR for this repo. They would need the info in a texual way to know what are the requirements for the PR to pass :)

**Comment by ADmad** [comment]
I think the messages set for `message` keys is what will be shown to users.

File: .varci.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      The broken link is: {{ diff_links.broken.0 }}
+    when:
+      - action = "opened" or action = "reopened"
+      - count(diff_links.broken) > 0
+      - files has "README.md"
+
+  link_in_description:
+    name: "the URL matches the URL in the PR's body"
+    events: [ pull_request ]
+    comment: >
```
Comment (ADmad) [inline .varci.yml]: I think this key should also be `message` like others.

Comment (jadb) [inline .varci.yml]: Fixed on master. The `message` key was deprecated (but still supported). Updated all to `comment`.

**Pull request merged**